### PR TITLE
System.AggregateException F# snippets

### PR DESCRIPTION
**Pull request by albert-du**
## Summary

Describe your changes here.

Adds F# code snippets for System.AggregateException

### CODE REVIEW

**Comment by albert-du** [comment]
@baronfel @dsyme could you review this please?

File: samples/snippets/fsharp/VS_Snippets_CLR_System/system.aggregateexception.class/fs/exception1.fs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+let getAllFiles str =
+    // Should throw an UnauthorizedAccessException exception.
+    System.IO.Directory.GetFiles(str, "*.txt", System.IO.SearchOption.AllDirectories)
+
+// Get a folder path whose directories should throw an UnauthorizedAccessException.
+let path =
+    Environment.SpecialFolder.UserProfile
+    |> Environment.GetFolderPath
+    |> Directory.GetParent
+    |> fun x -> x.FullName
```
Comment (dsyme) [inline samples/snippets/fsharp/VS_Snippets_CLR_System/system.aggregateexception.class/fs/exception1.fs:15]: In general please don't use `|> fun x -> ...` in beginner code snippets.  The style is OK but confusing for beginners.  Just use a let binding, e.g. 

```fsharp
let path =
    let parent =
        Environment.SpecialFolder.UserProfile
        |> Environment.GetFolderPath
        |> Directory.GetParent
    parent.FullName
```

File: samples/snippets/fsharp/VS_Snippets_CLR_System/system.aggregateexception.class/fs/exception1.fs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+// Use this line to throw an exception that is not handled.
+// let task1 = Task<string []>.Factory.StartNew(fun () -> raise (IndexOutOfRangeException()) )
+let task1 = Task.Factory.StartNew(fun () -> getAllFiles (path))
+
+try
```
Comment (dsyme) [inline samples/snippets/fsharp/VS_Snippets_CLR_System/system.aggregateexception.class/fs/exception1.fs:21]: Put this kind of large code snippet involving a try/with in a function of its own

**Review by dsyme — APPROVED** [review]
Just two minor comments, marking as approved

**Comment by albert-du** [comment]
👍

**Review by BillWagner — APPROVED** [review]
Thanks @albert-du and @dsyme 

I'll :shipit: as soon as the build completes.

**Pull request merged**